src/content/en/updates/2017/03/performant-expand-and-collapse.md

```diff
@@ -0,0 +1,294 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: When making expand and collapse effects you can use scale transforms to keep your UI performant.
+
+{# wf_updated_on: 2017-03-23 #}
+{# wf_published_on: 2017-03-23 #}
+{# wf_tags: performance #}
+{# wf_featured_image: /web/updates/images/2017/03/performant-expand-and-collapse/featured-image.jpg #}
+{# wf_featured_snippet: When making expand and collapse effects you can use scale transforms to keep your UI performant. #}
+
+
+# Building performant expand & collapse animations {: .page-title }
+
+{% include "web/_shared/contributors/paullewis.html" %}
+{% include "web/_shared/contributors/smcgruer.html" %}
+
+**TL;DR - use scale transforms when animating clips. You can prevent the children from being
+stretched and skewed during the animation by counter-scaling them.**
+
+Previously we’ve posted updates on how to create performant 
+[parallax effects](/web/updates/2016/12/performant-parallaxing) and 
+[infinite scrollers](/web/updates/2016/07/infinite-scroller). In this 
+post we’re going to look over what’s involved if you want performant clip animations. If you want 
+to see a demo, check out the [Sample UI Elements GitHub repo](https://github.com/GoogleChrome/ui-element-samples/tree/gh-pages/animated-clip).
+
+Take, for example, an expanding menu:
+
+<div style="display: flex; justify-content: center">
+  <video src="/web/updates/images/2017/03/performant-expand-and-collapse/menu.mp4" loop autoplay>
+  </video>
+</div>
+
+Some options for building this are more performant than others.
+
+## Bad: Animating width and height on a container element
+
+You could imagine using a bit of CSS to animate the width and height on the container element.
+
+    .menu {
+      overflow: hidden;
+      width: 350px;
+      height: 600px;
+      transition: width 600ms ease-out, height 600ms ease-out;
+    }
+
+    .menu--collapsed {
+      width: 200px;
+      height: 60px;
+    }
+
+The immediate problem with this approach is that it requires animating `width` and `height`. 
+These properties require calculating layout and paint the results on every frame of the animation, 
+which can be very expensive, and will typically cause you to miss out on 60fps. If that’s news to 
+you then read our 
+[Google Web Fundamentals Rendering Performance](/web/fundamentals/performance/rendering/)
```

Comment:
You could probably just say 'Rendering Performance'. I don't think you need a five word title.